Discovery/Discovery/Presentation/WebPrograms/ProgramWebviewView.swift

```diff
@@ -114,3 +130,25 @@ public struct ProgramWebviewView: View {
         .animation(.default, value: viewModel.showError)
     }
 }
+
+#if DEBUG
+//swiftlint:disable all
```

Comment:
why do we need to disable swiftlint here?

Reply:
Thanks for bringing that to my attention. I've removed it.